db/migrate/20240718195012_add_person_to_form_answers.rb

```diff
@@ -0,0 +1,5 @@
+class AddPersonToFormAnswers < ActiveRecord::Migration[7.1]
+  def change
+    add_reference :form_answers, :person, null: false, foreign_key: true
```

Comment:
Why did you add a reference to person? If we look at the associations for FormAnswer it's just Organization and FormSubmission.